src/tigerbeetle/cli.zig

```diff
@@ -109,6 +109,102 @@ const CliArgs = union(enum) {
         seed: ?u64 = null,
     },
 
+    // Experimental: the interface is subject to change.
+    inspect: union(enum) {
+        superblock: struct {
+            positional: struct { path: []const u8 },
+        },
+        wal: struct {
+            slot: ?usize = null,
+            positional: struct { path: []const u8 },
+        },
+        replies: struct {
+            slot: ?usize = null,
+            superblock_copy: u8 = 0,
+            positional: struct { path: []const u8 },
+        },
+        grid: struct {
+            block: ?u64 = null,
+            superblock_copy: u8 = 0,
+            positional: struct { path: []const u8 },
+        },
+        manifest: struct {
+            superblock_copy: u8 = 0,
+            positional: struct { path: []const u8 },
+        },
+        tables: struct {
+            superblock_copy: u8 = 0,
+            tree: []const u8,
+            level: ?u6 = null,
+            positional: struct { path: []const u8 },
+        },
```

Comment:
Nit, but I'd probably go for `inspect --superblock <path>` rather than subcommands. It's unclear how exactly this'll will evolve, and a single command with a bunch of flags is more flexible.

Eg, I can imagine that at some point we'll want to add a `--format=json` flag, and adding that too all subcommands would be a chore. 

It also feels like

```
$ insperct --superblock --wal <datafile>
```

could be a thing.

Reply:
:thinking: I like subcommands b/c each subcommand can have an independent set of flags for validation. e.g. `wal` has `--slot`, but `superblock` doesn't.

For something like `--format=json`, we could handle that the same way as `path`:
```
const path = switch (command) {
    inline else => |args| args.positional.path,
};
```